content/docs/gherkin/reference.md

```diff
@@ -2,14 +2,16 @@
 title: Gherkin Reference
 subtitle: "Cucumber syntax: Given, When, Then"
 ---
-Gherkin uses a set of special keywords to give structure and meaning to
+Gherkin uses a set of special [keywords](#keywords) to give structure and meaning to
 executable specifications. Each keyword is translated to many spoken languages;
 in this reference we'll use English.
 
 Most lines in a Gherkin document start with one of the [keywords](#keywords).
 
-Comment lines are allowed anywhere in the file. They begin with zero or more spaces,
-followed by a hash sign (`#`) and some text. Comments do have to start on a new line.
+A comment must start on a new line. Comment lines are allowed anywhere in the file. They begin with zero or more spaces,
```

Comment:
this sentence contradicts itself.

```
When I do something # a comment
```
As an example is ambiguous by this statement.

If you leave it as mentioned before. Or perhaps amend it to something like `Comments are only permitted at the start of a new line, anywhere in the feature file`